charts/gitops-server/values.yaml

```diff
@@ -0,0 +1,106 @@
+# Default values for chart.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+  # FIXME check the app name
+  repository: ghcr.io/weaveworks/wego-app
+  pullPolicy: IfNotPresent
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: "v0.6.2"
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+
+logLevel: info
+# Additional arguments to pass in
+# additionalArgs: []
+
+# To enable enterprise OIDC:
+# envVars:
+# - name: WEAVE_GITOPS_AUTH_ENABLED
+#   value: true
+
+serviceAccount:
+  # Specifies whether a service account should be created
+  create: true
+  # Annotations to add to the service account
+  annotations: {}
+  # The name of the service account to use.
+  # If not set and create is true, a name is generated using the fullname template
+  name: ""
+
+rbac:
+  # Specifies whether the clusterRole & binding to the service account should be created
+  create: true
+  # List of resources that the service account can impersonate
+  impersonationResourceNames: []
+  # Limit the type of principal that can be impersonated
+  # impersonationResources: [ "users", "groups" ]
+  # Specify whether additional permissions should be granted to the service account
+  # in order to work with weave-gitops enterprise
+  viewSecrets: []
+
+# This should not be used in production
+testUser:
+  # Whether the wego-admin user should be created.
+  # If you use this make sure you add it to rbac.impersonationResourceNames.
+  create: false
+  # helm will base64 encode these
+  username: not-production-worthy
+  password: seriously-change-me
+
+podAnnotations: {}
+
+podSecurityContext: {}
+  # fsGroup: 2000
+
+securityContext: {}
+  # capabilities:
+  #   drop:
+  #   - ALL
+  # readOnlyRootFilesystem: true
```

Comment:
just for context, if users make the pod fs rootfs read only, i think the profiles mechanism will fail as it writes a disk-based cache

Reply:
Is the profiles mechanism supposed to always be available? I was unclear as to whether it was an enterprise only feature. (this is default copy but once its use is clarified I can update & fix this).

Reply:
See this thread here: https://weaveworks.slack.com/archives/C03244W0C8H/p1646387603804719

Reply:
(It has no resolution and I am still confused, but just for context)

Reply:
https://github.com/weaveworks/weave-gitops/issues/1785

Reply:
I'm unsure what it does but it feels like it's separate to what the rest of the ui is (currently) doing

Reply:
yep 💯  Core v2 has no real concept of what profiles are yet. so let's shuffle it away until they come into scope